modules/shortcodes/class.filter-embedded-html-objects.php

```diff
@@ -242,6 +242,10 @@ public static function maybe_create_links( $string ) {
 	}
 
 	static function get_attrs( $html ) {
+		if ( ! ( function_exists( 'libxml_use_internal_errors' ) && function_exists( 'simplexml_load_string' ) ) ) {
+			trigger_error( 'PHP has been compiled without the libxml or SimpleXML libraries. Please recompile PHP with these libraries.' );
```

Comment:
I like Core's language more, yes, can you please change it here? Otherwise this is great, thank you!